spring-integration-core/src/main/java/org/springframework/integration/endpoint/AbstractMessageSource.java

```diff
@@ -131,8 +131,8 @@ public void setLoggingEnabled(boolean loggingEnabled) {
 	}
 
 	@Override
+	@Deprecated
```

Comment:
@garyrussell ,

if you are still here, I would like to ask about this deprecation.

Should we do the same for the super method in the `MessageSourceMetrics` at all? And therefore remove it altogether from here since it is a `default` one anyway there ?

Reply:
Ah; yes, we should; thanks.